core/dbt/parser/schemas.py

```diff
@@ -786,6 +798,9 @@ class NodePatchParser(
     def parse_patch(
         self, block: TargetBlock[NodeTarget], refs: ParserRef
     ) -> None:
+        # We're not passing the ParsedNodePatch around anymore, so we
+        # could possibly skip creating one. Leaving here for now for
+        # code consistency.
```

Comment:
imo, less code is better in this case.

Reply:
I always like removing code :) But it's more changes than I want to do at this point. We can fix next time this is touched.

Reply:
I'm also in favor of a teeny tiny PR to make this change 🙂